### PR TITLE
Version bump

### PR DESCRIPTION
**Pull request by Anubhavjain786**

### CODE REVIEW

**Review by imagekitio — CHANGES_REQUESTED** [review]

File: sdk/tests/sdk-tests/ik-image.component.spec.ts [hunk 1]
```diff
@@ -42,7 +42,7 @@ describe("IkImageComponent", () => {
     iKService = new ImagekitService({
       urlEndpoint: "https://ik.imagekit.io/company",
       publicKey: "abc",
-      authenticationEndpoint: "http://example.com/auth"
+      // authenticationEndpoint: "http://example.com/auth"
```
[the remark this answers is not in the document]
Comment (NikhilSejwal7) [reply sdk/tests/sdk-tests/ik-image.component.spec.ts]: Done

File: README.md [hunk 2]
```diff
@@ -11,6 +11,51 @@ ImageKit Angular SDK allows you to resize, optimize, deliver, and upload images
 
 ImageKit is complete media storage, optimization, and transformation solution that comes with an image and video CDN. It can be integrated with your existing infrastructure - storage like AWS S3, web servers, your CDN, and custom domain names, allowing you to deliver optimized images in minutes with minimal code changes.
 
+## Changelog - SDK Version 2.0.0
```
[the remark this answers is not in the document]
Comment (NikhilSejwal7) [reply README.md]: Fixed

File: sdk/tests/test-apps/sample-server/package-lock.json [hunk 3]
```diff
@@ -1,83 +1,112 @@
...
+      }
+    },
+    "node_modules/abbrev": {
       "version": "1.1.1",
       "resolved": "https://registry.npmjs.org/abbrev/-/abbrev-1.1.1.tgz",
       "integrity": "sha512-nne9/IiQ/hzIhY6pdDnbBtz7DjPTKrY00P/zvPSm5pOFkl6xuGrGnXn/VtTNNfNtAfZ9/1RtehkszU9qcTii0Q==",
       "dev": true
     },
-    "accepts": {
+    "node_modules/accepts": {
```
[the remark this answers is not in the document]
Comment (NikhilSejwal7) [reply sdk/tests/test-apps/sample-server/package-lock.json]: Fixed

File: sdk/tests/sdk-tests/ik-upload.component.spec.ts [hunk 4]
```diff
@@ -1,561 +1,622 @@
...
-    const options: IkUploadComponentOptions = {
-      file: dummyFile,
-      fileName: 'dummyFile',
-      onSuccess: comp.onSuccess
-    }
-    comp.handleUploadResponse(undefined, 'success', options, xhr, progressCb);
-    expect(hasTrackedProgress).toBeTruthy();
-  });
-});
+// import { ComponentFixture, TestBed } from '@angular/core/testing';
```
[the remark this answers is not in the document]
Comment (NikhilSejwal7) [reply sdk/tests/sdk-tests/ik-upload.component.spec.ts]: Uncommented and fixed test cases that were failing

**Pull request closed without merging**